events/static/js/app/AddCompanyWindow.js

```diff
@@ -0,0 +1,212 @@
+import React from 'react';
+import Dialog from 'material-ui/Dialog';
+import AutoComplete from 'material-ui/AutoComplete';
+import FlatButton from 'material-ui/FlatButton';
+import RaisedButton from 'material-ui/RaisedButton';
+import CompanyTextField from './CompanyTextField';
+import axios from 'axios';
+export default class AddCompanyWindow extends React.Component {
+
+    constructor(props) {
+        super(props);
+        this.state = {
```

Comment:
Just side note.

don't you supposed to use `getInitialState`?

> #### Avoiding State
> The use of state should be as limited as it can get. When you use state, you run the risk of introducing a number of (sometimes subtle) errors in the behaviour and rendering of your components.
https://medium.com/react-tutorials/react-state-14a6d4f736f5#.qezxakc85